spec/luassert_helper.lua

```diff
@@ -1,18 +1,28 @@
 local busted = require('busted')
-local assert = require("luassert")
+local assert = require('luassert.assert')
 local util = require ('luassert.util')
 local say = require('say')
 
 do -- set up reverting stubs
-  local snapshot
+  local state = require("luassert.state")
 
-  busted.before_each(function()
-    snapshot = assert:snapshot()
-  end)
+  local function snapshot()
+    state.snapshot()
+    return nil, true
```

Comment:
Why do all these methods need to return `nil, true` ?

Reply:
Otherwise the callback chain gets cancelled. Going to write a comment / helper.